infer_structcol/model.py

```diff
@@ -84,34 +88,46 @@ def calc_resid_spect(spect1, spect2):
                     sigma_r = sigma_eff_r, transmittance = residual_t, 
                     sigma_t = sigma_eff_t)
 
-def calc_log_prior(theta):
+def calc_log_prior(theta, theta_range):
```

Comment:
This change means that the hard coded min/max values at the top of the file are no longer used anywhere in this file. Maybe that means there's a better place to put them? I'm not sure...